manifests/package/debian.pp

```diff
@@ -42,11 +45,13 @@
           default => $repo_source,
         }
         apt::source { 'nginx':
-          location     => $mainline_repo_source,
-          repos        => 'nginx',
-          key          => { 'id' => '573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62' },
-          release      => $release,
-          architecture => $facts['os']['architecture'],
+          location            => $mainline_repo_source,
+          repos               => 'nginx',
+          key                 => {
+            'name'   => 'nginx',
+            'source' => 'https://nginx.org/keys/nginx_signing.key',
+          },          release => $release,
+          architecture        => $facts['os']['architecture'],
```

Comment:
Formatting got a bit messed up here. `release` should  be on its own line, and there are too many spaces before the arrows.